share/docker/Dockerfile

```diff
@@ -92,6 +92,11 @@ RUN apt-get -qqy install --no-install-recommends libzstd-dev \
                                                  php-mbstring \
                                                  php-dev
 
+RUN wget -q https://www.openssl.org/source/openssl-1.1.1q.tar.gz && \
```

Comment:
```
wget http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
dpkg -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb
rm -f libssl1.1_1.1.1f-1ubuntu2_amd64.deb
```
This should work as well, without recompiling openssl1.1.

Reply:
Yes, the compiling takes quite a while. We should add an if-else to make this work for the `arm64` arch's as well

Reply:
This should work for multi arch:
```
ARCH="$(dpkg --print-architecture)"
LIBSSL_DEB="libssl1.1_1.1.1f-1ubuntu2_$ARCH.deb"
if [ "$ARCH" == "amd64" ] || [ "$ARCH" == "i386" ]; then
    LIBSSL_URL="http://security.ubuntu.com/ubuntu/pool/main/o/openssl/$LIBSSL_DEB"
else
    LIBSSL_URL="http://ports.ubuntu.com/ubuntu-ports/pool/main/o/openssl/$LIBSSL_DEB"
fi
wget "$LIBSSL_URL"
dpkg -i "$LIBSSL_DEB"
rm -f "$LIBSSL_DEB"
```

Reply:
Beautiful, thank you!